pyatv/protocols/dmap/__init__.py

```diff
@@ -612,12 +612,12 @@ def hscp_service_handler(
     return name, service
 
 
-def scan() -> Mapping[str, ScanHandler]:
+def scan() -> Mapping[str, ScanHandlerDeviceInfoName]:
     """Return handlers used for scanning."""
     return {
-        "_appletv-v2._tcp.local": homesharing_service_handler,
-        "_touch-able._tcp.local": dmap_service_handler,
-        "_hscp._tcp.local": hscp_service_handler,
+        "_appletv-v2._tcp.local": (homesharing_service_handler, lambda _: None),
```

Comment:
I guess these services should do some kind of extraction as well and not just return `None`?

Reply:
These devices don't have enough information in the name to derive the device info name so `None` is expected

Reply:
Ok, great! Then we are all set for now.